### DISCUSSION TITLE
My css style wont show up everytime i host my page on here

### DISCUSSION THREAD

**Discussion by umyesiamcandy** [community/community#140225] [New to GitHub]
Okay so to start, I create my repository I export my style.CSS and index.HTML from codepen and move it onto the main branch files. then i go to settings > pages > put my main into my root and EVERYTIME it keeps coming up like this with NO css style on it
https://ayoung588.github.io/wunderlust/
I dont know what to do and everything just seems insanely complicated
https://github.com/Ayoung588/wunderlust. this is my repository

**Comment by feijoes** [comment]
what do you mean? For me the css works fine
![imagen](https://github.com/user-attachments/assets/22cee746-008c-4670-a94f-e349560c437f)
